### PR TITLE
Remove support for PG11

### PR DESCRIPTION
**Pull request by svenklemm**
Disable-check: commit-count

### CODE REVIEW

**Review by pmwkaa — APPROVED** [review]
Looks good

File: tsl/src/nodes/gapfill/exec.c [hunk 1]
```diff
@@ -23,18 +24,12 @@
...
-#else
-#include <optimizer/optimizer.h>
-#endif
-
 #include "nodes/gapfill/gapfill.h"
 #include "nodes/gapfill/locf.h"
 #include "nodes/gapfill/interpolate.h"
 #include "nodes/gapfill/exec.h"
 #include "time_bucket.h"
+#include "compat.h"
```
Comment (mkindahl) [inline tsl/src/nodes/gapfill/exec.c:32]: Not sure why you're keeping it here: you've removed it everywhere else. What function is it needed for?

**Review by mkindahl — APPROVED** [review]
LGTM. I saw several opportunities for optimization that you *didn't* do, which made my job as a reviewer a lot easier, so thank you for that.

**Review by mfundul — APPROVED** [review]

File: tsl/src/nodes/gapfill/exec.c (hunk 1 above)
Comment (svenklemm) [reply tsl/src/nodes/gapfill/exec.c:32]: needed for TTSOpsVirtualP

Comment (mkindahl) [reply tsl/src/nodes/gapfill/exec.c:32]: Ok.

**Pull request merged**